docs/Changelog.md

```diff
@@ -8990,6 +8990,86 @@ This version of the operator has been available since version 9 of the default O
+   
+  In contrast to standard n-gram extraction, here, the indexes of extracting an n-gram from the original 
+  sequence are not necessarily consecutive numbers. The discontinuity between indexes are controlled by the number of skips.  
+  If the number of skips is 2, we should skip two tokens when scanning through the original sequence. 
+  Let's consider an example. Assume that input sequence is [94, 17, 36, 12, 28] and the number of skips is 2. 
+  The associated 2-grams are [94, 12] and [17, 28] respectively indexed by [0, 3] and [1, 4]. 
+  If the number of skips becomes 0, the 2-grams generated are [94, 17], [17, 36], [36, 12], [12, 28] 
+  indexed by [0, 1], [1, 2], [2, 3], [3, 4], respectively.
+  
+  The output vector stores the count of each n-gram; 
+  Y[i] indicates the times that the i-th n-gram is found. The attribute ngram_indexes is used to determine the mapping 
+  between index i and the corresponding n-gram. If pool_int64s is [94 , 17 ,17, 36], ngram_indexes is [1, 0],
+  ngram_counts=[0, 0], then the Y[0] (first element in Y) and Y[1] (second element in Y) are the counts of [17, 36] and [94, 17],
+  respectively. An n-gram which cannot be found in pool_strings/pool_int64s should be ignored and has no effect on the output. 
+  Note that we may consider all skips up to S when generating the n-grams. 
+   
+  The examples used above are true if mode is "TF". If mode is "IDF", all the counts larger than 1 would be truncated to 1 and 
+  the i-th element in weights would be used to scale (by multiplication) the count of the i-th n-gram in pool. If mode is "TFIDF", 
+  this operator first computes the counts of all n-grams and then scale them by the associated values in the weights attribute. 
+   
+  Only one of pool_strings and pool_int64s can be set. If pool_int64s is set, the input should be an integer tensor. 
+  If pool_strings is set, the input must be a string tensor. 
+
+#### Version
+
+This version of the operator has been available since version 9 of the default ONNX operator set.
+
+#### Attributes
+
+<dl>
+<dt><tt>max_gram_length</tt> : int (required)</dt>
+<dd>Maximum n-gram length. If this value is 3, 3-grams will be used to generate the output.</dd>
+<dt><tt>max_skip_count</tt> : int (required)</dt>
+<dd>Maximum number of items (integers/strings) to be skipped when constructing an n-gram from X.If max_skip_count=1, min_gram_length=2, max_gram_length=3, this operator may generate 2-gramswith skip_count=0 and skip_count=1, and 3-grams with skip_count=0 and skip_count=1</dd>
+<dt><tt>min_gram_length</tt> : int (required)</dt>
+<dd>Minimum n-gram length. If this value is 2 and max_gram_length is 3, output may contain counts of 2-grams and 3-grams.</dd>
+<dt><tt>mode</tt> : string (required)</dt>
+<dd>The weighting criteria. It can be one of "TF" (term frequency),"IDF" (inverse document frequency), and "TFIDF" (the combination of TF and IDF)</dd>
```

Comment:
```suggestion
<dd>The weighting criteria. It can be one of "TF" (term frequency),"IDF" (inverse document frequency), and "TFIDF" (the combination of "TF" and "IDF")</dd>
```